binaries/cli/src/main.rs

```diff
@@ -127,10 +151,26 @@ fn main() {
 }
 
 fn run() -> eyre::Result<()> {
-    #[cfg(feature = "tracing")]
-    set_up_tracing("dora-cli").context("failed to set up tracing subscriber")?;
     let args = Args::parse();
 
+    #[cfg(feature = "tracing")]
+    match args.command {
+        Command::Daemon { .. } => {
+            set_up_tracing("dora-daemon").context("failed to set up tracing subscriber")?;
+        }
+        Command::Runtime => {
+            // Do not set the runtime in the cli.
+        }
+        Command::Coordinator { .. } => {
+            set_up_tracing("dora-coordinator").context("failed to set up tracing subscriber")?;
+        }
+        _ => {
+            set_up_tracing("dora-cli").context("failed to set up tracing subscriber")?;
+        }
+    };
+
+    let ctrlc_events = set_up_ctrlc_handler()?;
```

Comment:
We should do this only when the daemon or coordinator is launched. For the CLI, we want to keep the standard ctrl-c handling.